backend/src/controllers/city.controller.ts

```diff
@@ -1,54 +1,72 @@
-import { City, Master, Order } from './../types';
+import { City, CityAttributes } from './../models/city.model';
 import { Request, Response } from 'express';
-import db from '../db';
 import { validate } from '../validate';
 
 export default class CityController {
     async addCity(req: Request, res: Response): Promise<Response> {
         const error: string = await validate(req.body, ['name']);
-
         if (error) return res.status(400).json(error);
 
-        const { name }: City = req.body;
-        const city: City[] = (await db.query(`SELECT * FROM addCity($1)`, [name])).rows;
-        return res.status(201).json(city[0]);
+        const { name }: CityAttributes = req.body;
+        try {
+            const city = await City.create({ name });
+            return res.status(201).json(city);
+        } catch (e) {
+            return res.status(500).json(e);
+        }
     }
     async getCities(req: Request, res: Response): Promise<Response> {
-        const cities: City[] = (await db.query('SELECT * FROM city ORDER BY id')).rows;
-        return res.status(200).json(cities);
+        try {
+            const cities = await City.findAll({
+                order: [
+                    ['id', 'ASC']
+                ]
+            });
+            return res.status(200).json(cities);
+        } catch (e) {
+            return res.status(500).json(e);
+        }
     }
     async getCityById(req: Request, res: Response): Promise<Response> {
         const error: string = await validate(req.params, ['id']);
-
         if (error) return res.status(400).json(error);
 
         const id: number = Number(req.params.id);
```

Comment:
Instead of use Number you can just type +
const id: number = +req.params.id;
Also you don't need type number. Typescript understand that it will be number type because you use '+' or 'Number'